host-configs/rzansel-blueos_3_ppc64le_ib_p9-clang@8.0.1_nvcc_xlf.cmake

```diff
@@ -24,15 +24,15 @@ set(CMAKE_Fortran_COMPILER "/usr/tce/packages/xl/xl-2019.12.23/bin/xlf2003" CACH
 #------------------------------------------------------------------------------
 
 # Root directory for generated TPLs
-set(TPL_ROOT "/usr/WS1/axom/libs/blueos_3_ppc64le_ib_p9/2020_05_04_19_55_16/clang-8.0.1_nvcc_xlf" CACHE PATH "")
+set(TPL_ROOT "/usr/WS1/axom/libs/blueos_3_ppc64le_ib_p9/2020_06_15_22_33_17/clang-8.0.1_nvcc_xlf" CACHE PATH "")
 
 set(CONDUIT_DIR "${TPL_ROOT}/conduit-0.5.1" CACHE PATH "")
 
-set(MFEM_DIR "${TPL_ROOT}/mfem-4.0" CACHE PATH "")
+set(MFEM_DIR "${TPL_ROOT}/mfem-4.1.0" CACHE PATH "")
 
 set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.21" CACHE PATH "")
 
-# Lua not built
+set(LUA_DIR "${TPL_ROOT}/lua-5.3.5" CACHE PATH "")
```

Comment:
Are we sure that we want to use `lua@5.3`?
E.g. are there specific feature of recent lua versions that you think we need?

Some codes are stuck at lua@5.1 for compatibility with `LuaJIT`  (https://luajit.org/faq.html)

Reply:
We are tracking whatever was the highest release in spack. There shouldn't be anything specific I am doing in my Inlet branch that is specific.  I do hear there are some type changes (like int32 vs int64) you can do depending on how you build it and what version you are on.